src/domain/occurrences/fields/OccurrenceAttendedField.tsx

```diff
@@ -54,8 +54,4 @@ const OccurrenceAttendedField = () => {
   );
 };
 
-OccurrenceAttendedField.defaultProps = {
-  label: 'enrolments.fields.attended.label',
-};
-
```

Comment:
Does removing this affect functionality? If it does, should the functionality be added back in some other way? Or is the functionality not needed?